### PR TITLE
Allow generating random numbers in nick on page load

### PR DESCRIPTION
**Pull request by xPaw**
Very useful in public mode. No need to rely on `nick in use` to generate random.

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -521,7 +521,7 @@ function getClientConfiguration() {
 	config.themes = themes.getAll();
 
 	if (config.displayNetwork) {
-		config.defaults = Helper.config.defaults;
+		config.defaults = _.clone(Helper.config.defaults);
```
Comment (astorije) [inline src/server.js]: We should not call that defaults anymore IMO. Not necessarily for your PR though.

File: defaults/config.js [hunk 2]
```diff
@@ -236,10 +236,13 @@ module.exports = {
...
 		//
+		// Percent sign (%) will be replaced into a random number from 0 to 9.
+		// For example, Guest%%% will become Guest123 on page load.
+		//
 		// @type     string
-		// @default  "lounge-user"
+		// @default  "thelounger%%"
 		//
-		nick: "lounge-user",
+		nick: "thelounger%%",
```
Comment (astorije) [inline defaults/config.js]: I'd prefer to have `thelounge-user%%` but whatever.

Comment (xPaw) [reply defaults/config.js]: We can do `TheLounge%%`

Comment (astorije) [reply defaults/config.js]: lol no thanks. `thelounge%%`?

**Pull request merged**